package.json

```diff
@@ -19,11 +19,11 @@
   "license": "MIT",
   "exports": {
     ".": {
+      "types": "./dist/index.d.ts",
       "import": "./dist/index.mjs",
       "require": "./dist/index.js",
       "browser": "./dist/index.mjs",
-      "node": "./dist/index.js",
-      "types": "./dist/index.d.ts"
+      "node": "./dist/index.js"
```

Comment:
Those `browser` and `node` conditions are likely **never** matched. I would advise you to remove them or reorder this because `package.json#exports` are **order-sensitive**